woocommerce/woocommerce-gutenberg-products-block
# Decode HTML entities when formatting Store API error messages

Fixes #5869 




#### Accessibility




#### Other Checks


### Screenshots

See #5869 

### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Add a callback to the wooocommerce_store_api_validate_cart_item action and throw a RouteException that includes HTML entities.
2. Open the Cart Block.
3. You should see a notice with the message you added to your Exception.
4. Go to the Checkout Block.
5. Attempt to place the order.
6. You should see the same message, with entities decoded.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).




### Performance Impact



### Changelog

> Decode HTML entities when formatting Store API error messages.
